src/app/App.js

```diff
@@ -568,6 +568,9 @@ class App extends EventEmitter {
 	 */
 	handleNavigateError_(path, nextScreen, err) {
 		console.log('Navigation error for [' + nextScreen + '] (' + err + ')');
+		if (err.isBlockedByCORS) {
```

Comment:
We don't route external domains, therefore CORS will never happen, although this error is to handle masked CORS that comes from a redirect from a URL of the same domain of the page. We can prob here rename to something more specific, e.g. `isBlockedByCORSAfterRedirect`.